src/clusterfuzz/_internal/bot/testcase_manager.py

```diff
@@ -534,7 +535,7 @@ def _do_run_testcase_and_return_result_in_queue(crash_queue,
       # Don't upload uninteresting testcases (no crash) or if there is no log to
       # correlate it with (not upload_output).
       if upload_output:
-        upload_testcase(file_path, log_time)
+        upload_testcase(file_path, log_time, None)
```

Comment:
wrong order of arguments?

Reply:
Youre right.
I think because of how many edge cases our code has there's no actual misbehavior resulting from this bug